### PR TITLE
Create Rule5-45.md

### PR DESCRIPTION
**Pull request by xingcx**

### CODE REVIEW

File: docs/section5/Rule5-45.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+
+# Envelope - Rule 5-45  
+
+**Rule ID:** 5-45  
+**Rule Description:** The  infiltration schedules are the same in the proposed RMR as in the baseline RMR.  
+**Rule Assertion:** B-RMR hourly fractions of infiltration:infiltration_multiplier_schedule_name = P-RMR hourly fractions of infiltration:infiltration_multiplier_schedule_name.
```
Comment (mkarpman) [inline docs/section5/Rule5-45.md:6]: Rule assertion should be that the schedules are the same, not that the names of the schedules are the same

**Review by mkarpman — APPROVED** [review]

File: docs/section5/Rule5-45.md (hunk 1 above)
Comment (xingcx) [reply docs/section5/Rule5-45.md:6]: Corrected to infiltration.multiplier_schedule.

File: docs/section5/Rule5-45.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+        - Get zone infiltration in P_RMR: `infiltration_p = zone_p.infiltration`
+
+          **Rule Assertion:**  
+
+          - Case 1: If zone infiltration schedule in P_RMR matches that in B_RMR: `if compare_schedules(infiltration_b.multiplier_schedule, infiltration_p.multiplier_schedule) == "EQUAL": PASS`  
+
+          - Case 2: Else: `Else: FAIL`
+
+**[Back](../_toc.md)**
```
Comment (supriyagoel) [inline docs/section5/Rule5-45.md:41]: @mkarpman would it be useful to add an output of the comparison result out here. we havent yet written the comparison function but we could provide information on number of hrs the schedules are the same, P schedule is > B or B schedule is > P.

**Review by supriyagoel — COMMENTED** [review]
question on whether some output of schedule comparison should be provided in FAIL case.

File: docs/section5/Rule5-45.md (hunk 2 above)
Comment (mkarpman) [reply docs/section5/Rule5-45.md:41]: I agree that it will be useful as it will determine severity of the issue.

**Review by mkarpman — APPROVED** [review]

File: docs/section5/Rule5-45.md (hunk 2 above)
Comment (mkarpman) [reply docs/section5/Rule5-45.md:39]: Please update the message to say "Baseline and proposed infiltration schedules are not the same. The ratio of Baseline EFLH to Proposed EFLH is equal to {compare_schedules_result_dictionary[EFLH_DIFFERENCE]}

**Review by mkarpman — APPROVED** [review]

File: docs/section5/Rule5-45.md (hunk 2 above)
Comment (xingcx) [reply docs/section5/Rule5-45.md:39]: Updated.

**Review by mkarpman — APPROVED** [review]

**Review by supriyagoel — APPROVED** [review]

**Pull request merged**